Dockerfile

```diff
@@ -13,14 +12,22 @@ WORKDIR /app
 
 ENV PYTHONUNBUFFERED=1
 
+# Install system dependencies for mysqlclient and netcat
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    default-libmysqlclient-dev \
+    pkg-config \
+    netcat-openbsd && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
The installation of system dependencies is done correctly, but it's important to ensure that all necessary packages for your application are included. Double-check if `build-essential` and `default-libmysqlclient-dev` are required for your application, as they can increase the image size.